Web/registration.js

```diff
@@ -0,0 +1,98 @@
+// JavaScript Document
+function validate(){
+	
+	var pwd= document.getElementById('password').value; 
+	var re_pwd= document.getElementById('re_password').value; 
+	if(pwd!=re_pwd)
+		{
+			alert("Not same password and confirm password");
+			return false;
+			
+		}
+	var age_value=document.getElementById('age').value;
+	var filter = /^[0-9]+(.[0-9]{1,2})?$/;
```

Comment:
ACTION: Check your filter expression is working properly. Try with comma, semi-column. It is not working